### PR TITLE
Sass edits apr 7 2020

### PR DESCRIPTION
**Pull request by jhogue**
Various things. Notes in each commit message about what I am adding or editing and sometimes a Why.

### CODE REVIEW

File: sass/media/_images.scss [hunk 1]
```diff
@@ -0,0 +1,33 @@
+// Post Thumbnails — Featured images
+.has-post-thumbnail > .post-thumbnail {
+	// Pull out into the parent wrapper padding
+	margin: 0 -1rem;
+
+  @media ($xlarge-up) {
+		margin: 0 -2.5rem;
```
Comment (kangabell) [inline sass/media/_images.scss:7]: indent

Comment (jhogue) [reply sass/media/_images.scss:7]: FUUUUUCK

File: sass/site/primary/_posts-and-pages.scss [hunk 2]
```diff
@@ -149,6 +149,37 @@
...
+.card.type-vehicle.tag-sold,
+.tag-sold .entry-content .ngg-gallery-thumbnail {
+	
+	a {
+  	display: block;
+  	position: relative;
+  	overflow: hidden;
+  	
+  	&::after {
+    	background-color: #b5161c;
```
Comment (kangabell) [inline sass/site/primary/_posts-and-pages.scss]: indent

File: sass/site/primary/_posts-and-pages.scss [hunk 3]
```diff
@@ -149,6 +149,37 @@
...
+    	color: #fff;
+    	font-size: 1.125rem;
+    	font-weight: bolder;
```
Comment (kangabell) [inline sass/site/primary/_posts-and-pages.scss]: this is not a weight used on this site

File: sass/site/primary/_posts-and-pages.scss [hunk 4]
```diff
@@ -149,6 +149,37 @@
...
+.tag-sold .entry-content .ngg-gallery-thumbnail {
+	
+	a {
+  	display: block;
+  	position: relative;
+  	overflow: hidden;
+  	
+  	&::after {
+    	background-color: #b5161c;
+    	color: #fff;
```
Comment (kangabell) [inline sass/site/primary/_posts-and-pages.scss]: use a variable

File: sass/site/primary/_posts-and-pages.scss [hunk 5]
```diff
@@ -149,6 +149,37 @@
...
+    	font-size: 1.125rem;
+    	font-weight: bolder;
+    	content: 'SOLD';
+    	letter-spacing: 1px;
```
Comment (kangabell) [inline sass/site/primary/_posts-and-pages.scss]: should be em's not px

File: sass/site/primary/_posts-and-pages.scss [hunk 6]
```diff
@@ -149,6 +149,37 @@
...
+    	padding: 0.375rem 6rem;
+    	position: absolute;
+    	top: 1.5rem;
+    	right: -5rem;
+    	transform: rotate(45deg);
+    }
+  }
+}
+
+.tag-sold .entry-content .ngg-gallery-thumbnail a::after {
```
Comment (kangabell) [inline sass/site/primary/_posts-and-pages.scss:178]: this is very high specificity. I'm wondering if it's really necessary, but it's also fine.

File: sass/elements/_elements.scss [hunk 7]
```diff
@@ -40,65 +40,67 @@ figure {
...
-		.textwidget {
-			align-content: center;
-			display: flex;
-			flex-wrap: wrap;
-			justify-content: space-around;
-		}
-	}
+  display: none;
+  visibility: hidden;
+	//@include full-width;
```
Comment (kangabell) [inline sass/elements/_elements.scss]: what? why are we hiding this?

File: sass/site/primary/_posts-and-pages.scss (hunk 6 above)
Comment (jhogue) [reply sass/site/primary/_posts-and-pages.scss:178]: I know, i'm not crazy about it but I didn't also want to go through the work of creating a specific template or PHP handler for the sold tag

File: sass/elements/_elements.scss (hunk 7 above)
Comment (jhogue) [reply sass/elements/_elements.scss]: Because clients are gonna client

Comment (kangabell) [reply sass/elements/_elements.scss]: hm  okay... Can you please remove from the DOM. And then just remove this entire block of text. I don't leave commented-out code in my repo. It should just be removed.

File: sass/site/primary/_posts-and-pages.scss (hunk 6 above)
Comment (kangabell) [reply sass/site/primary/_posts-and-pages.scss:178]: hmm yeah I think PHP would be more appropriate than a pseudo-element, but I see why you don't wanna bother.
I'd think `.tag-sold .ngg-gallery-thumbnail a::after` would work to simplify the selector, but 🤷‍♀

File: sass/typography/_typography.scss [hunk 8]
```diff
@@ -2,7 +2,7 @@ html {
 	-moz-osx-font-smoothing: grayscale;
 	-webkit-font-smoothing: antialiased;
 	font-smoothing: antialiased;
-	font-size: 16px;
+	font-size: 100%;
```
Comment (kangabell) [inline sass/typography/_typography.scss:5]: disagree. tested earlier version (`16px`) and it does not override the browser setting (Firefox & Chrome). are you actually seeing this issue?

Comment (kangabell) [reply sass/typography/_typography.scss:5]: nevermind, was testing in an environment that was overriding the 16px oops. it's fine.

File: sass/elements/_elements.scss (hunk 7 above)
Comment (jhogue) [reply sass/elements/_elements.scss]: Took a different approach. See latest push.

**Pull request merged**